src/components/work/details/details.utils.js

```diff
@@ -0,0 +1,316 @@
+  const mixed = [...main, ...spoken, ...subtitles];
+  // filter out duplicates
+  return [...new Set(mixed)].join(", ");
+}
+
+/**
+ * Genre and form from given work.
+ * @param work
+ * @returns {string}
+ *  comma seperated string.
+ */
+function parseGenreAndForm(work) {
+  const genreForm = work?.genreAndForm || [];
+  return genreForm.join(", ");
+}
+
+/**
+ * Physical description of given manifestations.
+ * @param manifestation
+ * @returns {string | undefined}
+ *  description summary (ies) seperated by space
+ */
+function parsePhysicalDescriptions(manifestation) {
+  return manifestation?.physicalDescriptions
+    ?.map((description) => description.summary)
+    .join(" ");
+}
+
+/**
+ * Creators AND contributors for given manifestation
+ * @param manifestation
+ * @returns {false|JSX.Element}
+ */
+function parseContributors(manifestation) {
+  const contributors = [
+    ...(manifestation?.creators || []),
+    ...(manifestation?.contributors || []),
+  ];
+
+  return (
+    contributors?.length > 0 && (
+      <ParsedCreatorsOrContributors
+        creatorsOrContributors={contributors}
+        Tag={CreatorContributorTextHelper}
+      />
+    )
+  );
+}
+
+/**
+ * Contributors for given manifestation.
+ * For some cases we need to split creators and contributors (eg movies)
+ * @param manifestation
+ * @returns {{}|null}
+ */
+function parseMovieContributors(manifestation) {
+  const actors = manifestation?.contributors?.filter((cont) =>
+    cont?.roles?.find((rol) => rol.functionCode === "act")
+  );
+
+  const others =
+    manifestation?.contributors?.manifestation?.contributors?.filter((cont) =>
+      cont?.roles?.find((rol) => rol.functionCode !== "act")
+    );
+  const actorslabel =
+    actors?.length > 1
+      ? actors?.[0]?.roles?.[0]?.function?.plural
+      : actors?.[0]?.roles?.[0]?.function?.singular;
+
+  const returnObject = {};
+
+  // check if any values
+  if (isEmpty(actors) && isEmpty(others)) {
+    return null;
+  }
+
+  returnObject[actorslabel] = actors?.length > 0 ? actors : null;
+  returnObject["ophav"] = others?.length > 0 ? others : null;
+
+  return returnObject;
+}
+
+/**
+ * Creators for given manifestation
+ * @param manifestation
+ * @returns {*|null}
+ */
+function parseMovieCreators(manifestation) {
+  return manifestation?.creators || null;
+}
+
+/**
+ * Map a single person ({disploy{roles[{function, functioncode}]}}
+ * @param person
+ * @returns {*}
+ *  a string "disploy (function)" .. eg "ebbe fisk (instruktør)"
+ */
+function parsePersonAndFunction(person) {
+  const display = person?.display;
+  const roles = person?.roles?.map((role) => role?.function?.singular || "");
+  return display + (roles.length > 0 ? " (" + roles.join(", ") + ")" : "");
+}
+
+/**
+ * jsxParser for movie creators
+ * @param values
+ * @param skeleton
+ * @returns {unknown[]}
+ *  jsx element to be parsed by react
+ * @constructor
+ */
+function MovieCreatorValues({ values, skeleton }) {
```

Comment:
Du kan defaulte til `values = []` for at undgå at skulle tjekke.

Som det er nu: Hvorfor kan der både være `values` og `skeleton` samtidig?